bin/mapbox-geostats

```diff
@@ -27,6 +27,8 @@ const help = `
     --tile-stats-values-limit Limit the number of unique attribute values to
                               report. Default: 50. If exceeded, show min and
                               max instead.
+    --add-languages Save all the languages from the vector data into the
+                    metadata languages array. Default: true.
```

Comment:
Save all the languages from the cector data into the tilestats as languages array.